test/buildenv_check.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
See jedi-system-update see how we test that all things we need from the environment are correctly installed

Reply:
Updates, please review :) .